tox.ini

```diff
@@ -5,7 +5,9 @@ envlist = py37,py38,py39,py310
 [testenv:{unit,py37,py38,py39,py310,py}]
 description = unit testing
 skip_install = true
-passenv = DBT_* PYTEST_ADDOPTS
+passenv =
```

Comment:
I think we'll want to backport just the `passenv` changes in this file

Reply:
PRs for this change are  up in PRs across the repo 🙏